src/dashboard/src/components/administration/urls.py

```diff
@@ -28,6 +28,8 @@
     (r'dips/contentdm/$', 'contentdm_dips'),
     (r'sources/$', 'sources'),
     (r'storage/$', 'storage'),
+    (r'usage/$', 'usage'),
+    (r'usage/clear/(?P<dir_id>\w+)/$', 'usage_clear'),
```

Comment:
I agree - being able to do automation would be useful, but do we want to give remote access to deleting things?  Especially for uploadedDIPs, which may contain DIPs that (despite the name :P) haven't actually been sent elsewhere and are awaiting manual download.